wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupPasswordViewController.swift

```diff
@@ -71,24 +72,40 @@ final class BackupPasswordViewController: UIViewController {
     }
 
     private func setupViews() {
-        view.backgroundColor = SemanticColors.View.backgroundDefault
+        view.backgroundColor = ViewColors.backgroundDefault
         passwordRulesLabel.text = PasswordRuleSet.localizedErrorMessage
 
         [passwordView, subtitleLabel, passwordRulesLabel].forEach {
             view.addSubview($0)
             $0.translatesAutoresizingMaskIntoConstraints = false
         }
 
-        passwordView.placeholder = HistoryBackup.Password.placeholder.capitalized
+        passwordView.placeholder = HistoryBackup.Password.placeholder
         passwordView.accessibilityIdentifier = "password input"
         passwordView.accessibilityHint = PasswordRuleSet.localizedErrorMessage
         passwordView.returnKeyType = .done
         passwordView.isSecureTextEntry = true
         passwordView.delegate = self
         passwordView.textColor = LabelColors.textSectionHeader
         passwordView.backgroundColor = ViewColors.backgroundUserCell
-        let attributes: [NSAttributedString.Key: Any] = [.foregroundColor: SemanticColors.SearchBar.textInputViewPlaceholder,
-                                                        .font: FontSpec.smallRegularFont.font!]
+
+        // Set a fixed font size
+        let fixedFont = UIFont.systemFont(ofSize: 17)
+        passwordView.font = fixedFont
+
+        // Enable large content viewer
```

Comment:
I actually didn't know about this feature until this PR. Thanks!

But I'm wondering if this is really what we want in this case - documentation from apple states:

>Don’t use the large content viewer as a replacement for proper Dynamic Type support. For example, Dynamic Type allows items in a list to grow or shrink vertically to accommodate the user’s preferred font size. Rely on the large content viewer only in situations where items must remain small due to unavoidable design constraints. For example, buttons in a tab bar remain small to leave more room for the main app content.

I don't have a strong feeling about this but thought it maybe worth discussing.

Reply:
I remember seeing somewhere that's recommended for even UITextFields. For now though I can't find the source, hence I reverted the commit. https://github.com/wireapp/wire-ios/commit/7498791dd2b1b5001260e26e8e12e5de828038ad

Reply:
👍 Maybe let's keep an eye out if we find the recommendation again. It's not always best to rely on Apples documentation!